### DISCUSSION TITLE
Your update has been rejected. Reason: Wallet Testing

### DISCUSSION THREAD

**Discussion by RAcbd** [WalletConnect/walletconnect-monorepo#3645] [Developer Support]
Project ID: 4ebd64e9dd6b28533e82e9f166990666

"Could not verify if testing has been done on this wallet. As we do not have available V2 metric data for Sign. Please test your integration with our test dapp and submit for approval if successful. Link: https://lab.web3modal.com/"

Our project has been getting rejected over and over after just an image update (We updated our Logo listing for wallets). We we're already listed as of Feb 14, 2023 and just go delisted due to this. We've already implemented V2 way back and have successfully tested on https://lab.web3modal.com/ that sign works. There isn't enough data to go by with this response on what we need to do.

Any help would be appreciated.

**Comment by RAcbd** [comment]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/22793295/c0f4360e-4962-4581-ab3f-a91ea6c73ba2)

**Comment by RAcbd** [comment]
Still need help

**Reply by JonathanConn** [reply]
Hello, sorry for the delay, looking into this now 👋

**Reply by JonathanConn** [reply]
Looks like you were rejected because there are no sign v2 metrics on our end
You can see them by going to your project in cloud and selecting the metrics tab

Can you confirm you have sign v2 implemented?
And if so, your project is using the correct project id (4ebd64e9dd6b28533e82e9f166990666) ?
